PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/webapp/webappconfig/WebAppRunState.java

```diff
@@ -151,8 +151,8 @@ private void openWebAppInBrowser(String url, RunProcessHandler processHandler) {
 
     @Override
     protected void onSuccess(WebAppBase result, @NotNull RunProcessHandler processHandler) {
-        if (AzureUIRefreshCore.listeners != null) {
-            AzureUIRefreshCore.execute(new AzureUIRefreshEvent(AzureUIRefreshEvent.EventType.REFRESH, null));
+        if (webAppSettingModel.isCreatingNew() && AzureUIRefreshCore.listeners != null) {
+            AzureUIRefreshCore.execute(new AzureUIRefreshEvent(AzureUIRefreshEvent.EventType.REFRESH,null));
```

Comment:
AzureUIRefreshEvent(AzureUIRefreshEvent.EventType.REFRESH,null));
Missing space between "REFRESH" and "null"

Reply:
got it, resolved